### ISSUE TITLE
Called horse is always invisible and can't be interacted with

### ISSUE THREAD

**Issue by theLime98** [jodeks-datapacks/Call-Your-Horse#1]
As the title says, when I call my horse in my singleplayer world it will always be invisible and act as if it's not there. To fix it I can reload the game but it's quite annoying to have to reload every time.

**Comment by Jodekq** [comment]
Yup thats a bug I encountered too. Sadly its a minecraft issue and I can't really much. But if you don't call your horse from too far away it shouldn't be like that.

**Comment by Jodekq** [comment]
But this should be fixed in 1.21.1 tho
https://youtu.be/wQmG7wj7yYw?si=_Ye2kLzvfArDhfXD&t=276
